### PR TITLE
Renaming: owner -> prjOwner, projectUuid -> uuid

### PR DESCRIPTION
**Pull request by odeimaiz**
In order to avoid confusion and double checks, renamed:
- owner -> prjOwner
- projectUuid -> uuid

### CODE REVIEW

File: services/web/client/source/resource/qxapp/project-v0.0.1.json [hunk 1]
```diff
@@ -5,19 +5,19 @@
   "type": "object",
   "additionalProperties": false,
```
Comment (sanderegg) [inline services/web/client/source/resource/qxapp/project-v0.0.1.json]: this file is duplicated in apihub. -> needs to be updated in apihub or the webserver will not be in sync
we need a mechanism to prevent this duplication: the webserver should provide the jsonschema to the webclient

Comment (odeimaiz) [reply services/web/client/source/resource/qxapp/project-v0.0.1.json]: Deleted and case created #485

File: api/specs/webserver/v0/components/schemas/project-v0.0.1-converted.yaml [hunk 2]
```diff
@@ -3,18 +3,18 @@ description: Description of a simcore project
 type: object
 additionalProperties: false
 required:
-  - projectUuid
+  - uuid
   - name
   - description
   - notes
-  - owner
+  - prjOwner
```
Comment (pcrespov) [inline api/specs/webserver/v0/components/schemas/project-v0.0.1-converted.yaml]: I wonder why this change. You remove prefix project to ``uuid`` but adds it to ``owner``.
Is ``owner`` a reserved word?

Comment (odeimaiz) [reply api/specs/webserver/v0/components/schemas/project-v0.0.1-converted.yaml]: Correct, "owner" is kind of reserved word in the qx-compiler we are using at the moment. I think that the latest compiler already avoids using it though.
